app/src/main/res/values/strings.xml

```diff
@@ -41,6 +41,16 @@ You’ll also find exercises at the bottom to challenge yourself at varying leve
         <item>silver</item>
         <item>pillow</item>
     </string-array>
+    <string-array name="real_sentences">
+        <item>The cat sits on the mat</item>
+        <item>Birds fly across the sky</item>
+        <item>She walks to the park</item>
+        <item>It rains on summer days</item>
+        <item>The stars shine at night</item>
+        <item>Dogs bark in the yard</item>
+        <item>The sun rises every day</item>
+        <item>A child plays with toys</item>
+    </string-array>
```

Comment:
While you’ve added real_sentences in strings.xml, it would be great to add tests that validate their correct usage and display within ExerciseScreenHard. For example, you could test whether sentences are loaded and displayed. It's a good thing to put this in strings.xml, well done !